src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/TextDescriptor.php

```diff
@@ -337,9 +338,13 @@ protected function describeContainerDefinition(Definition $definition, array $op
                     $argument = $argument->getValues()[0];
                 }
                 if ($argument instanceof Reference) {
-                    $argumentsInformation[] = sprintf('Service(%s)', (string) $argument);
+                    $argumentsInformation[] = sprintf('Service (%s)', (string) $argument);
                 } elseif ($argument instanceof IteratorArgument) {
-                    $argumentsInformation[] = sprintf('Iterator (%d element(s))', \count($argument->getValues()));
+                    if ($argument instanceof TaggedIteratorArgument) {
+                        $argumentsInformation[] = sprintf('Tagged Iterator for "%s"%s', $argument->getTag(), $options['is_debug'] ? '': sprintf(' (%d element(s))', \count($argument->getValues())));
+                    } else {
+                        $argumentsInformation[] = sprintf('Iterator%s', $options['is_debug'] ? '' : sprintf(' (%d element(s))', \count($argument->getValues())));
```

Comment:
do we really need the ternary on this line?

Reply:
You are right we don't need the ternary. The array of services given for the Iterator is always filled. Will revert it.

Reply:
done